src/Sprite/ImageMeta.h

```diff
@@ -29,6 +29,11 @@ struct ImageMeta {
 	uint32_t width; // Width of image in pixels
 	ImageType type;
 	uint16_t paletteIndex;
+
+	bool IsShadow() const
+	{
+		return type.bShadow != 0;
+	}
```

Comment:
Considering all fields in the `struct` are public, maybe we should name the field `isShadow` and access it directly.
```cpp
if (imageMeta.type.isShadow) { ... }
```

Reply:
I was also using a bool type for the helper function so it was easier to interpret pass / fail. The rename to isShadow sounds like a good idea either way though.

Reply:
Hmm, I hadn't considered the type. I suppose it's not possible to adjust the type without interfering with the bit field.